testing/streaming-e2e/bot/index.js

```diff
@@ -61,33 +80,25 @@ const myBot = new EchoBot();
 
 // Listen for incoming requests.
 server.post('/api/messages', (req, res) => {
-    adapter.processActivity(req, res, async (context) => {
-        // Route to main dialog.
-        await myBot.run(context);
-    });
+    adapter.process(req, res, (context) => myBot.run(context));
 });
 
-adapter.useNamedPipe(async (context) => {
-    await myBot.run(context);
-    },
-    process.env.APPSETTING_WEBSITE_SITE_NAME + '.directline'
+adapter.connectNamedPipe(
+    `${process.env.APPSETTING_WEBSITE_SITE_NAME}.directline`,
+    (context) => myBot.run(context),
```

Comment:
Should this be `async`, or does it not matter?

Reply:
[Not in this case 😄](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body). The return value is propagated because there are no braces.